test/quicer_SUITE.erl

```diff
@@ -366,222 +325,15 @@ tc_lib_re_registration(_Config) ->
   ok = quicer:reg_close(),
   ok = quicer:reg_close().
 
-tc_open_listener_inval_parm(Config) ->
-  Port = select_port(),
-  ?assertEqual({error, config_error, invalid_parameter},
-               quicer:listen(Port, [ {stream_recv_buffer_default, 1024} % too small
-                                   | default_listen_opts(Config)])),
-  ok.
-
-tc_open_listener_inval_cacertfile_1(Config) ->
-  Port = select_port(),
-  ?assertEqual({error, badarg},
-               quicer:listen(Port, [ {cacertfile, atom}
-                                   | default_listen_opts(Config)])),
-  ok.
-
-tc_open_listener_inval_cacertfile_2(Config) ->
-  Port = select_port(),
-  {error, badarg} = quicer:listen(Port, [ {cacertfile, [1,2,3,4]}
-                                | default_listen_opts(Config)]),
-  ok.
-
-tc_open_listener_inval_cacertfile_3(Config) ->
-  Port = select_port(),
-  ?assertEqual({error, badarg},
-               quicer:listen(Port, [ {cacertfile, [-1]}
-                                   | default_listen_opts(Config)])),
-  ok.
-
-tc_open_listener(Config) ->
-  Port = select_port(),
-  {ok, L} = quicer:listen(Port, default_listen_opts(Config)),
-  {ok, {_, Port}} = quicer:sockname(L),
-  {error, eaddrinuse} = gen_udp:open(Port),
-  ok = quicer:close_listener(L),
-  {ok, P} = gen_udp:open(Port),
-  ok = gen_udp:close(P),
-  ok.
-
-tc_open_listener_with_cert_password(Config) ->
-  Port = select_port(),
-  DataDir = ?config(data_dir, Config),
-  PasswordCerts = [ {certfile, filename:join(DataDir, "server-password.pem")}
-                  , {keyfile,  filename:join(DataDir, "server-password.key")}
-                  , {password, ?SERVER_KEY_PASSWORD}
-                  ],
-  {ok, L} = quicer:listen(Port, default_listen_opts(PasswordCerts ++ Config)),
-  quicer:close_listener(L),
-  ok.
-
-tc_open_listener_with_wrong_cert_password(Config) ->
+tc_open_listener_inval_reg(Config) ->
```

Comment:
Q: isn't this a listener test too?

Reply:
it should be but it is also testing a negative case when its dep (registration) is unavailable(closed). 

Actually, I moved it back here because due to the current limitation in msquic and quicer I could not close the global registration when I have lingering listener and connections which is created/used then forgotten in the suite. This tc must be run before any other test run otherwise it is deadlock.